go-controller/pkg/ovn/pods.go

```diff
@@ -184,7 +186,11 @@ func getRoutesGatewayIP(pod *kapi.Pod, gatewayIPnet *net.IPNet) ([]util.PodRoute
 			routes = append(routes, route)
 		}
 	} else {
-		gatewayIP = gatewayIPnet.IP
+		if podExternalGW == "" {
+			gatewayIP = gatewayIPnet.IP
+		} else {
+			gatewayIP = util.NextIP(util.NextIP(gatewayIPnet.IP))
```

Comment:
@rcarrillocruz can you use util.GetNodeWellKnownAddresses() here please?

Reply:
makes sense